announcementsAPI.json

```diff
@@ -288,6 +296,45 @@
         }
       }
     },
+    "/announcements/{announcementId}/like": {
+      "put": {
+        "tags": [
+          "AnnouncementsEndpoints"
+        ],
+        "summary": "Adds or deletes announcement from liked announcements depending on boolean value in parameters. Requires adopter role. Gets adopter id from auth claims.",
+        "parameters": [
+         {
+            "name": "announcementId",
+            "in": "path",
+            "required": true,
+            "style": "simple",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "isLiked",
+            "in": "query",
+            "required": true,
+            "style": "form",
+            "schema": {
+              "type": "boolean"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "404": {
+            "description": "Announcement with provided id does not exists"
+          }
+        }
```

Comment:
We can also have `403 forbidden` if i.e. shelter sends a request here. And I think we can have `400 bad request` as well, if there is no query parameter. That doesn't affect the 'positive' flow though, so it's not really important.

Reply:
Ok, I can add those responses as well